greenplumpython/col.py

```diff
@@ -35,8 +35,8 @@ def __init__(
     def column(self) -> "Column":
         return self._column
 
-    def serialize(self) -> str:
-        return f'({self.column.serialize()})."{self._field_name}"'
+    def _serialize(self) -> str:
+        return f'({self.column._serialize()})."{self._field_name}"'  # type: ignore
```

Comment:
Since `reportPrivateUsage = false` has been set, the line suppression is not needed I think?